### ISSUE TITLE
Excluding the identifier_name 'no' disables all exclusions

### ISSUE THREAD

**Issue by CraigSiemens** [realm/SwiftLint#1616]
Using version 0.19.0. Worked fine in 0.17.0.

When I exclude the identifier_name `no`, all other excluded variable names are ignored.

## Source
```swift
let a = 1
let no = 2
let b = 3
```

## .swiftlint.yml
```cml
identifier_name:
  excluded:
    - a
    - no #Commenting this line will cause a,b to be excluded
    - b
```

Using this swift lint file causes `a`, `no`, and `b` to be marked as violations.

```
$ swiftlint lint
Loading configuration from '.swiftlint.yml'
Linting Swift files in current working directory
Linting 'test.swift' (1/1)
/Users/craig/Desktop/SwiftLintTest/test.swift:1:1: error: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'a' (identifier_name)
/Users/craig/Desktop/SwiftLintTest/test.swift:2:1: warning: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'no' (identifier_name)
/Users/craig/Desktop/SwiftLintTest/test.swift:3:1: error: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'b' (identifier_name)
Done linting! Found 3 violations, 2 serious in 1 file.
```

Commenting out the `no` exclusion in the swift lint file allows `a`, and `b` to be ignored properly.

```
$ swiftlint lint
Loading configuration from '.swiftlint.yml'
Linting Swift files in current working directory
Linting 'test.swift' (1/1)
/Users/craig/Desktop/SwiftLintTest/test.swift:2:1: warning: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'no' (identifier_name)
Done linting! Found 1 violation, 0 serious in 1 file.
```

Also tried wrapping the names in the swift lint file in both double and single quotes but that had no effect.

**Comment by norio-nomura** [comment]
duplicate of #1424
